app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons = []
+    for person in people:
+        name = person["name"]
+        age = person["age"]
+        persons.append(Person(name, age))
```

Comment:
Consider using list comprehension to create the list of `Person` instances from the `people` list. This is recommended by the checklist to make the code more concise and Pythonic.